src/pages/new/docs/configuration/client-auth/index.vue

```diff
@@ -0,0 +1,89 @@
+- truststorePassword: Password of the trust store file
+- certificateVerification: "required", "optional" or "none" (default) to enforce, allow or disable client authentication respectively
+
+If certificateVerification is set to "required" or "optional" a client certificate will be requested from the incoming request and validated against the CA certificates in the truststore.
+The leaf certificates (the actual partner certificates) are not required to added to the truststore.
+
+To enforce client authentication for all incoming requests set certificateVerification to "required".
+Be aware that this will require anyone sending a request to NEXUSe2e using the port to send a client certificate. This includes requests to the UI in the browser that are made using that port.
+It is the most secure setting and recommended, if it can be ensured that all requests will include a client certificate.
+
+If certificateVerification is set to "optional", a client certificate will still be requested and validated against the truststore, but not providing a certificate will also be accepted.
+This setting is recommended in most cases because it allows enforcing client authentication for specific partners without enforcing it for all requests.
+
+Example server.xml https connector configuration:
+`)
+
+const markdown2 = marked(`
+    <Connector port="11443" protocol="org.apache.coyote.http11.Http11NioProtocol"
+               maxThreads="150" SSLEnabled="true" secure="true" scheme="https">
+        <UpgradeProtocol className="org.apache.coyote.http2.Http2Protocol" />
+        <SSLHostConfig truststoreFile="conf/truststore.jks" truststorePassword="changeit" certificateVerification="optional">
+            <Certificate certificateKeystoreFile="conf/certificate.p12"
+                         certificateKeystorePassword="nexuse2e"
+                         type="RSA" />
+        </SSLHostConfig>
+    </Connector>
+`)
+
+const markdown3 = marked(`
+### NEXUSe2e configuration
+
+If client authentication is enabled in Tomcat ("required" or "optional"), it can be enforced for a specific [Route](#/new/docs/concepts/routes) by specifying an Inflow Pipeline that includes the ClientCertRulePipelet for the external partner on the route.
+For incoming requests from that partner that use this route a client certificate will then be required an validated against the certificates configured for that partner.
+
+Example route configuration:
+`)
+
+const markdown4 = marked(`
+## Outbound
+
+To configure NEXUSe2e to send a client certificate to the partner with an outbound messages, a client certificate container can be configured for the internal partner of a specific [Route](#/new/docs/concepts/routes).
+The client certificate container should contain your own current certificate that is known to the partner.
+While multiple certificates can be added to every certificate container, only one certificate should be active at any time within the container.
```

Comment:
In einem ClientCertContainer ist immer nur ein Zertifikate aktiv. Aktiviert man ein anderes wird das bestehende automatisch deaktiviert.